HW-2025-07-07/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Start</title>
```

Comment:
nit: Я думаю тут все ж таки краще прописати щось типу такого
```suggestion
    <title>Volodymyr Mishchuk | CV</title>
```